docs/en/registry-catalogue.rst

```diff
@@ -0,0 +1,338 @@
+.. include:: ../common/common_definitions.rst
+
+
+.. _registry_catalogue.rst:
+
+Digital Credentials Catalogue
++++++++++++++++++++++++++++++++++++++++++++
+
+The Digital Credentials Catalogue is the registry of all available Digital Credentials recognized within the IT-Wallet ecosystem. It is published by the Trust Anchor and publicly available by all Entities through a specialized Federation endpoint. It acts as a single reference point for all actors involved in the process of issuing, verifying and using Digital Credentials. 
+
+The Digital Credential Catalogue aims to:
+
+  1. Facilitate Digital Credential discovery for Users.
+  2. Standardize the technical and functional description of Digital Credentials.
+  3. Enable interoperability between different Issuers and Relying Parties.
+  4. Simplify the integration process for Wallet Providers and Relying Parties.
+  5. Ensure trust in the ecosystem through certified information.
+  6. Provide transparency on the ecosystem of available Digital Credentials.
+
+
+The main Entities involved in the Digital Credential Catalogue are: 
+
+  - **Trust Anchor**: It manages and maintains the Digital Credential Catalogue, guaranteeing its authenticity and integrity.
+  - **Supervisory Body**: It interacts with the Trust Anchor and the Digital Credential Catalogue to monitor the registration phase ensuring security and privacy according to national/European regulations, keeping all the information reliable and updated.
+  - **Digital Credential Issuers**: The entities authorized to issue Digital Credentials, registering them in the Catalogue.
+  - **Relying Parties**: They use the Digital Credential Catalogue to gather all the information needed about the Digital Credentials they intend to request during the presentation phase.
+  - **Wallet Providers**: They access the Digital Credential Catalogue to identify the available Digital Credentials and to retrieve all necessary information for integrating them into their Wallet Solutions.
+  - **Users**: The citizens who indirectly use the Digital Credentials Catalogue through their Wallet Instances to discover and request Digital Credentials.
+  - **Authentic Sources**: The Entities that hold the original data that is attested in the Digital Credentials. They provide support to Issuers in registering the Digital Credentials in the Catalogue. 
```

Comment:
do we really need to define again these main actors, once again, here? :-)

Reply:
It describes how the entities interact with the catalogue. this information is not included in the define terms.